vswitchd/bridge.c

```diff
@@ -3547,6 +3746,18 @@ bridge_create(const struct ovsrec_bridge *br_cfg)
 
     hmap_init(&br->mappings);
     hmap_insert(&all_bridges, &br->node, hash_string(br->name, 0));
+
+#if defined(P4OVS)
+    /* TODO: Implement a better logic for unique bridge ID of type uint8. */
+    if (last_p4_bridge_id_used <= MAX_P4_BRIDGE_ID) {
+        br->p4_bridge_id = last_p4_bridge_id_used++;
+        VLOG_DBG("Assigned unique P4 bridge ID of: %d, for bridge: %s",
+                    br->p4_bridge_id, br->name);
+    } else {
+        VLOG_WARN("Unable to assign unique P4 bridge ID for bridge: %s, reached"
```

Comment:
This block will never be executed with the current logic. Please reevaluate the logic to ensure that it operates as intended.